workflow/automation/execution_scripts/cybershake_progress.py

```diff
@@ -122,7 +108,7 @@ def get_new_progress_df(root_dir, runs_dir, faults_dict, mgmtdb: MgmtDB):
 
```

Comment:
104-107 Here @jasonmotha?

Reply:
Yeah, that's the closest we could make a comment - your way is better than mine.